src/calliope/backend/__init__.py

```diff
@@ -5,23 +5,30 @@
 import xarray as xr
 
 from calliope.backend.gurobi_backend_model import GurobiBackendModel
-from calliope.backend.latex_backend_model import MathDocumentation
+from calliope.backend.latex_backend_model import (
+    ALLOWED_MATH_FILE_FORMATS,
+    LatexBackendModel,
+)
 from calliope.backend.parsing import ParsedBackendComponent
 from calliope.backend.pyomo_backend_model import PyomoBackendModel
 from calliope.exceptions import BackendError
+from calliope.preprocess import ModelMath
 
-MODEL_BACKENDS = ("pyomo",)
+MODEL_BACKENDS = ("pyomo", "gurobi")
```

Comment:
In the tests for this `__init__` I am referring to this directly. Kind of a catch 22: either you forget to add it here, or you forget to add it to the test. I like this one because it's more visible.

The real backend tests do not use this tuple, so it's not a big issue.

Reply:
Also, this global is just so users can see the available backend options. Gurobi would not fail even if you remove it.

We can remove, if you wish